src/server/services/wms/qgswmsrenderer.cpp

```diff
@@ -2837,6 +2838,25 @@ namespace QgsWms
     if ( scaleDenominator > 0 )
       legendModel->setLegendFilterByScale( scaleDenominator );
 
+    // countSymbolFeatures is launched at the createLayerTreeModelLegendNodes
+    // with showFeatureCount property
+    if ( showFeatureCount )
+    {
+      Q_FOREACH ( QgsMapLayer *ml, layers )
```

Comment:
For loop instead of Q_FOREACH https://github.com/qgis/QGIS-Enhancement-Proposals/issues/103 ? ;-)

Reply:
done